### ISSUE TITLE
Add Environment Setup instructions

### ISSUE THREAD

**Issue by FedericoEsparza** [championer-org/championer_one#61]
Link to Environment Setup directly below the [`ChampionerOne`](https://github.com/championer-org/championer_one#championerone) in README.md.
 
However, actually put the full instructions for Windows, Mac, and Linux installations of Elixir 1.6, Phx 1.3, and Chromedriver above the [`Learn more about Championer`](https://github.com/championer-org/championer_one#learn-more-about-championer) section at the bottom of the README.md

**Comment by FedericoEsparza** [comment]
Mac Elixir and Phx 1.3 instructions:
`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
`brew install elixir`
`mix local.hex`
`mix local.rebar`
`mix archive.install https://github.com/phoenixframework/archives/raw/master/phx_new.ez`

**Comment by mattwr18** [comment]
Don't forget about `PostgreSQL`

**Comment by FedericoEsparza** [comment]
@mattwr18 Yep. Thanks. I'll post the `postgresql` instructions/[screenshots] here for whoever takes on the mac portion of this ticket.

https://postgresapp.com/
 
Here's the chromedriver install command for mac users:
`brew cask install chromedriver`

**Comment by FedericoEsparza** [comment]
Add nvm and npm instructions:
You can follow instructions here: https://github.com/creationix/nvm
or if you're on a mac:
Install nvm: 
run `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash`
add `source ~/.bashrc` to `~/.bash_profile`
Then run `source ~/.bash_profile `
`nvm install v8.9.4`
`nvm use v8.9.4`
`npm i -g npm@6.1.0`

**Comment by FedericoEsparza** [comment]
`chromedriver` installation instructions from the `travis.yml` file for linux users:
```
  - export CHROMEDRIVER_VERSION=`curl -s http://chromedriver.storage.googleapis.com/LATEST_RELEASE`
  - curl -L -O "http://chromedriver.storage.googleapis.com/${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip"
  - unzip chromedriver_linux64.zip
  - sudo chmod +x chromedriver
  - sudo mv chromedriver /usr/local/bin
```

**Comment by FedericoEsparza** [comment]
We need to doublecheck the versions for the PR for chromedriver: https://github.com/championer-org/championer_one/pull/93/files

**Comment by dkuku** [comment]
@FedericoEsparza I'll do this

**Comment by FedericoEsparza** [comment]
@dkuku Thanks, Daniel.
